content/docs/get-started/azure/modify-program.md

```diff
@@ -64,14 +64,14 @@ EOT
 
 {{% /choosable %}}
 
-Now that you have your new `index.html` with some content, you can enable static website support, upload `index.html` to a storage container, and retrieve a public URL through the use of resource properties. These properties can be used to define dependencies between related resources or to retrieve property values for further processing.
-
-To see how this all works, open the program file in your IDE or text editor and add the following the lines right after the storage account creation.
+Now that you have your new `index.html` with some content, enable static website support, upload `index.html` to a storage container, and retrieve a public URL through the use of resource properties. These properties can be used to define dependencies between related resources or to retrieve property values for further processing.
 
 {{< chooser language "typescript,python,go,csharp" / >}}
 
 {{% choosable language typescript %}}
 
+To start, open `index.ts` in your IDE or text editor and add the following right after the storage account creation:
```

Comment:
i wonder if we need to say IDE or text editor? could we just say open, then add?